netcore/Makefile.am

```diff
@@ -8,14 +8,32 @@ NETCORETESTS_VERSION := 4.6.0-preview5.19209.12
 # Extracted MicrosoftNETCoreAppVersion from https://github.com/dotnet/coreclr/blob/master/eng/Versions.props#L13
 NETCOREAPP_VERSION := 3.0.0-preview5-27609-17
 
-NETCORESDK_FILE := dotnet-runtime-$(NETCOREAPP_VERSION)-@RID@.tar.gz
+if HOST_WIN32
+NETCORESDK_EXT = zip
+UNZIPCMD = python -c "import zipfile,sys; zipfile.ZipFile(sys.argv[1], 'r').extractall()"
```

Comment:
I like the anonymous temporary. :)

Reply:
Might as well use .zip+python as the one portable solution? :)

Reply:
Only .tar.gz packages are available for Linux/macOS.

Reply:
Ok. xz will be much smaller as well, than any of them.

Reply:
@jaykrell True, but that's not our call. It downloads pre-packaged .NET Core runtime from https://github.com/dotnet/core-sdk/blob/master/README.md#installers-and-binaries and those are only available in one format per platform.